### ISSUE TITLE
> follow the dataset I write. https://github.com/lingtengqiu/Open-PIFuhd/blob/master/engineer/datasets/RPDataset.py, you could simply set camera matrix when u test

### ISSUE THREAD

**Issue by sahilshukla3003** [lingtengqiu/Open-PIFuhd#26]
> follow the dataset I write. https://github.com/lingtengqiu/Open-PIFuhd/blob/master/engineer/datasets/RPDataset.py, you could simply set camera matrix when u test
Thank you for your reply. I want to ask you one more question. During the training, whether the loss function you used is only mse

_Originally posted by @hanchaoyuan in https://github.com/lingtengqiu/Open-PIFuhd/issues/18#issuecomment-1350725544_
            

hii did you got the solution i am facing issue while trying to inference , os.fork was called ... , i don't know what is the issue I am trying to inference/test this project , i have passed single img at '/Garment/render_gen_1024_test/test.png'  and loaded the pretrained weights ....
![Screenshot from 2024-04-30 16-26-55](https://github.com/lingtengqiu/Open-PIFuhd/assets/124785012/ba6411e2-6c09-47b9-be69-25e299a1f002)
